libsolidity/ast/CallGraph.h

```diff
@@ -62,7 +62,7 @@ struct CallGraph
 	std::map<Node, std::set<Node, CompareByID>, CompareByID> edges;
 
 	/// Contracts that may get created with `new` by functions present in the graph.
```

Comment:
Please update the description. We could also change the name.

Reply:
Ah right, you mentioned the name change in the other PR. coming up.